src/Sylius/Bundle/ShopBundle/Resources/views/layout.html.twig

```diff
@@ -13,17 +13,10 @@
     {% endblock %}
 
     {% block stylesheets %}
-        <!--[if lt IE 9]>
-        <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
-        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-        <![endif]-->
-
-        {% include '@SyliusShop/_styles.html.twig' %}
-
-        {{ sonata_block_render_event('sylius.shop.layout.stylesheets') }}
+        {{ sylius_template_event('sylius.shop.layout.styles') }}
```

Comment:
Why `sylius.shop.layout.styles` not `sylius.shop.layout.stylesheets`?

Reply:
Its still exist https://github.com/Sylius/Sylius/pull/11126/files#diff-cd30ca237d9d140b0ba8594e7645c29eR99 in one section https://github.com/Sylius/Sylius/pull/11126/files#diff-cd30ca237d9d140b0ba8594e7645c29eR87-R99

Its combined to one event, but old event still exist,
This implementation is similar to : https://github.com/Sylius/Sylius/pull/11071/files#diff-f4ee3e9e9c146529dff5e394c42579b3L16-R16

https://github.com/Sylius/Sylius/pull/11071/files#diff-ad2a679f29cd8da7f22ee8746d7c6fafR251-R267